Fabulous.XamarinForms/samples/FabulousWeather/FabulousWeather/App.fs

```diff
@@ -0,0 +1,154 @@
+﻿namespace FabulousWeather
+
+open Fabulous
+open Fabulous.XamarinForms
+open Fabulous.XamarinForms.LiveUpdate
+open Xamarin.Forms
+
+module App =
+    type Operator = Add | Subtract | Multiply | Divide 
+
+    /// Represents a calculator button press
+    type Msg =
+        | Refresh
+
+    
+    type DataItem = {Name:string;Value:int}
+    type Model = {Temp:int;Items: DataItem list}
+
+    let initial = {Temp=61;Items=[
+        {Name="Pressure";Value=10}
+        {Name="UV Index";Value=3}
+        {Name="Wind Speed";Value=0}
+        {Name="Humidity";Value=65}
+        {Name="Min Temp";Value=50}
+        {Name="Max Temp";Value=80}
+        ]}
+     
+    let update msg model =
+        match msg with
+        | Refresh -> initial
+
+    let view (model: Model) dispatch =
+        let coldStartColor = Color.FromHex("#BDE3FA")
```

Comment:
It's a good idea to have reference colors! But declaring them here will reinstantiate them each time the view function is called.

Instead if you declare them outside the view function, they will be instantiated only once for the whole life of the application.

See for example (those are styling functions, but you can store your colors directly instead):
https://github.com/fsprojects/Fabulous/blob/6700d6f4143e88a60b3e1125159c9c3da795797a/Fabulous.XamarinForms/samples/Shell/Fabimals/Fabimals/App.fs#L16-L28